qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/customnavview/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -343,6 +347,15 @@ class MapboxNavigationViewCustomizedActivity : DrawerActivity() {
             viewModel.enableCompass,
             ::toggleEnableCompass
         )
+
+        MainScope().launch {
+            binding.navigationView.voiceInstructionsPlayerFlow.collect {
+                it.volume(SpeechVolume(0.1f))
+            }
+        }
```

Comment:
This is a good example on how to use it. We should add an example in our public examples repo to showcase how can one play additional custom instruction using Drop-In UI. I will cut a follow up ticket for that